pkg/nuctl/command/deploy.go

```diff
@@ -531,3 +546,23 @@ func (d *deployCommandeer) enrichConfigWithComplexArgs() error {
 
 	return nil
 }
+
+func (d *deployCommandeer) overrideHTTPTrigger(functionConfig *functionconfig.Config, serviceType v1.ServiceType) error {
+
+	for triggerName, trigger := range functionconfig.GetTriggersByKind(functionConfig.Spec.Triggers, "http") {
+		if trigger.Attributes == nil {
```

Comment:
that should not happen if `enrichFunctionConfig` is called, correct?

Reply:
we should either remove this block or enrich it with an empty attributes,
less error to handle, more fun for users